app/assets/stylesheets/components/_result-card.scss

```diff
@@ -44,3 +44,22 @@
 .app-c-result-card__link {
   @include govuk-font($size: 19, $weight: bold);
 }
+
+// stylelint-disable declaration-no-important
+@include govuk-media-query($media-type: print) {
+  .app-c-result-card {
+    break-inside: avoid;
+    border: 1pt solid $govuk-print-text-colour;
+    border-top-width: 1mm;
```

Comment:
Nitpick: does it need the slightly thicker top border? Having the border round it seems like highlight enough.

Reply:
I was emulating the screen styles, but you're right, we don't actually do this anywhere else for print as far as I know. I've removed it.